charts/panope/Chart.yaml

```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.7
+version: 0.2.0
```

Comment:
Sorry. I forgot to do a minor upgrade when I upgraded before.
Please upgrade to version 0.3.0.

Reply:
I upgraded.